src/components/todo/Todo.js

```diff
@@ -0,0 +1,68 @@
+import { Component } from 'react';
+import './todo.scss';
+
+export class Todo extends Component {
+  state = {
+    todos: [],
+    filter: ''
+  }
+
+  getPosts = () => {
+    fetch('https://jsonplaceholder.typicode.com/todos')
+      .then(data => data.json())
+      .then((posts) => {
+        this.originTodos = posts;
+        this.setState({ todos: this.originTodos });
+      });
+  }
+
+  componentDidMount() {
+    this.getPosts();
+  }
+
+  filterTodos = ({ target }) => {
+    if (target.value.length > 1) {
+      this.setState({ todos: this.originTodos.filter(({ title }) => title.includes(target.value)) });
+    } else {
+      this.setState({ todos: this.originTodos });
+    }
+
+    this.setState({ filter: target.value });
```

Comment:
Try to avoid calling `setState` few times:
```
    let todos = this.originTodos;

    if (target.value.length > 1) {
       todos = this.originTodos.filter(({ title }) => title.includes(target.value)) });
    }

    this.setState({ todos, filter: target.value });
```